### ISSUE TITLE
...no idea where to start!

### ISSUE THREAD

**Issue by savissimo** [hjson/vscode-hjson#9]
I'm sorry for the (semi-)comic title, but I'm really at a loss. I installed the VSCode extensions, and now if I create a .hjson file I can see the syntax highlighting working properly. Good. 

However, I have no idea what other features this extension has. For example, is there an auto-compile? Do I have to configure it? Do I have to start it? Is that even possible?

Is this extension just a language definition?

**Comment by dqsully** [comment]
Yes, this is only a language definition. Is there something else you want the extension to do?

**Comment by savissimo** [comment]
I've defined a task to compile an Hjson file to a JSON file with the same name - I'm not sure if this could be included in the extension, especially because it would have to be configurable with the path of the executable. But I haven't managed to make it work only when called on a .hjson file. 

However, what would be really good would be autobuild: a watcher on .hjson files that compiles them on save.

**Comment by JackLilhammers** [comment]
This is old, and it's very likely that the author of the post has already found a solution, but for converting between formats there is this extension: https://marketplace.visualstudio.com/items?itemName=petli-full.json-to-yaml-and-more

**Comment by Jeeppler** [comment]
A useful feature would be formatting.
